### ISSUE TITLE
Auto / Dense price granularity 

### ISSUE THREAD

**Issue by salvoaranzulla** [kmjennison/dfp-prebid-setup#27]
Hi,

Can you implement an option to create line items with "auto" or "dense" price granularity?

http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.setPriceGranularity

**Comment by kmjennison** [comment]
I agree this would be helpful. It's a known [limitation](https://github.com/kmjennison/dfp-prebid-setup#limitations) of this project, and the workaround now is to use the densest bucketing you need.

PR welcome. There's already a FIXME comment in settings.py to make `PREBID_PRICE_BUCKETS` an array.

**Comment by pboisso** [comment]
What I did to achieve the 'dense' price granularity is to run the script 3 times on the same order, but changing the PREBID_PRICE_BUCKETS each time.  Works like a charm.  Thanks so much for sharing such a great script !
